operator/config/default/kustomization.yaml

```diff
@@ -11,9 +11,11 @@ namePrefix: module-manager-
 # Labels to add to all resources and selectors.
 commonLabels:
   app.kubernetes.io/component: module-manager.kyma-project.io
-
-configurations:
-  - ./commonlabels_override.yaml
+  app.kubernetes.io/instance: module-manager-main
+  app.kubernetes.io/name: module-manager
+  app.kubernetes.io/created-by: kustomize
+  app.kubernetes.io/managed-by: kustomize
+  app.kubernetes.io/part-of: manual-deployment
```

Comment:
can we sync these labels with lifecycle-manager?

Reply:
the label name references from lifecycle-manager, or do you mean the `app.kubernetes.io/name` should have `kcp-` prefix? I think it should overwrite in management-control-plane (or after you migrated into this repo with different kustomization location, e.g: under `/kcp`).